app/bundles/CoreBundle/Test/AbstractMauticTestCase.php

```diff
@@ -105,7 +105,7 @@ protected function loadFixtureFiles(array $paths = [], bool $append = true): arr
 
     private function mockServices(): void
     {
-        self::$container->set('session', new Session(new FixedMockFileSessionStorage()));
+        //self::$container->set('session', new Session(new FixedMockFileSessionStorage()));
```

Comment:
@escopecz this line is not needed anymore, since `LiipFunctionalTestBundle` starts its own sessions, this line will break sessions. if you agree on the removal of this method I will proceed

Reply:
Let's remove these "mock services" completely. It was breaking my test last year so I refactored it to be toggleable. If we can remove it and all tests are passing then let's do that.